waspc/test/Test/Util.hs

```diff
@@ -0,0 +1,15 @@
+module Test.Util
+    ( posixToSystemFp
+    ) where
+
+import qualified Path as P
+import qualified System.FilePath as FP
+
+import Fixtures (fpRoot)
+
+posixToSystemFp :: FilePath -> FilePath
```

Comment:
Maybe I should put a comment here with examples.

Reply:
yes, would be nice!